test/web/ConsoleLineArgumentsParser.cs

```diff
@@ -11,15 +11,16 @@ public enum SocketEngineType
         EPoll,
         IOUring,
         IOUringTransport,
-        LinuxTransport
+        LinuxTransport,
+        DefaultSockets
```

Comment:
Maybe drop `Default` from the name?

Reply:
I'm unsure. In many cases such naming is used to indicate that a default option is being picked from the remaining enum members.

Reply:
I know what you mean by default, though it is not the default here.
Maybe: KestrelDefaultTransport?
Anyway, feel free to pick what you like and merge this.

Reply:
Ended up with the name `DefaultTransport`, since other variants are also missing the `Kestrel*` prefix.